docs/UserGuide.adoc

```diff
@@ -49,287 +49,289 @@ This app will not work with earlier versions of Java 8.
 +
 .  Download the latest `progresschecker.jar` link:{repoURL}/releases[here].
 .  Copy the file to the folder you want to use as the home folder for your ProgressChecker.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
++
+image::download3.png[width="790"]
++
+. After you click to open the progresschecker.jar app, you can see the following files will be automatically generated into the home folder where you copy jar file to.
```

Comment:
some grammar issue. "you can see the following files will be automatically generated into the home folder where you copy jar file to" might be rephrased as "you can see the following files automatically generated in(to) the home folder where you copied the jar file to"